### PR TITLE
🐛 Changes analytics notification example in amp.dev because the current example gives error when run locally 

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
In the analytics notification example, there are two examples. The current example gives a CORS issue on local repositories. As it calls some fake endpoints which may be only be whitelisted for some URLs. So, I think it is best to change it with the second example which works everywhere. I had a discussion on this with @micajuine-ho on issue no #24905. You can also check that out.

### CODE REVIEW

File: examples/analytics-notification.amp.html [hunk 1]
```diff
@@ -63,14 +63,14 @@
...
+    <amp-user-notification
+        layout=nodisplay
+        id="amp-user-notification6"
+        data-persist-dismissal="false">
+        This notification should ALWAYS show on every page visit.
+      <a href="#learn-more">Learn more.</a>
+      <button on="tap:amp-user-notification6.dismiss">Dismiss</button>
+    </amp-user-notification>
 
   <amp-analytics data-consent-notification-id="amp-user-notification1">
```
Comment (micajuine-ho) [inline examples/analytics-notification.amp.html]: Change this id to match the id of the `amp-user-notification`

Comment (ajitsinghkaler) [reply examples/analytics-notification.amp.html]: I changed the id back to the previous id

**Comment by ajitsinghkaler** [comment]
Can someone tell me what is the problem with this pull request why some checks are failing

**Comment by ajitsinghkaler** [comment]
@micajuine-ho Can you please help me get this reviewed.

File: examples/analytics-notification.amp.html [hunk 2]
```diff
@@ -63,14 +63,14 @@
...
-     <a class="btn" on="tap:amp-user-notification1.dismiss">I accept</a>
-  </amp-user-notification>
+    <amp-user-notification
+        layout=nodisplay
+        id="amp-user-notification1"
+        data-persist-dismissal="false">
+        This notification should ALWAYS show on every page visit.
+      <a href="#learn-more">Learn more.</a>
+      <button on="tap:amp-user-notification1.dismiss">Dismiss</button>
+    </amp-user-notification>
```
Comment (micajuine-ho) [inline examples/analytics-notification.amp.html]: remove extra tab

**Comment by ajitsinghkaler** [comment]
@micajuine-ho I've made the changes please have a look

**Comment by ajitsinghkaler** [comment]
@micajuine-ho I've made the changes I don't know why percy tests are failing

**Comment by micajuine-ho** [comment]
@ajitsinghkaler They seem to be flakey tests. I will force approve

**Comment by ajitsinghkaler** [comment]
Okay thanks

**Comment by ajitsinghkaler** [comment]
Okay thanks

**Pull request merged**